scripts/benchmarks/benchmark_utils.py

```diff
@@ -827,20 +829,16 @@ def run_forward():
             tvm_token_types = tvm.nd.array(token_types.asnumpy(), ctx=ctx)
             tvm_valid_length = tvm.nd.array(valid_length.asnumpy(), ctx=ctx)
 
-            def run_tvm_forward():
-                if 'roberta' in model_name or 'xlmr' in model_name:
-                    rt.set_input(data0=tvm_input_ids, data1=tvm_valid_length)
-                elif 'bart' in model_name:
-                    rt.set_input(data0=tvm_input_ids, data1=tvm_valid_length)
-                else:
-                    rt.set_input(data0=tvm_input_ids, data1=tvm_token_types,
-                                 data2=tvm_valid_length)
-                rt.run()
-                for i in range(rt.get_num_outputs()):
-                    out = rt.get_output(i)
-            # Warmup
-            timeit.repeat(run_tvm_forward, repeat=1, number=2)
-            runtimes = timeit.repeat(run_tvm_forward, repeat=self._repeat, number=3)
+            if 'roberta' in model_name or 'xlmr' in model_name:
+                rt.set_input(data0=tvm_input_ids, data1=tvm_valid_length)
+            elif 'bart' in model_name:
+                rt.set_input(data0=tvm_input_ids, data1=tvm_valid_length)
+            else:
+                rt.set_input(data0=tvm_input_ids, data1=tvm_token_types,
+                             data2=tvm_valid_length)
+            # ftimer returns a ProfileResult
+            ftimer = rt.module.time_evaluator("run", ctx, number=3, repeat=self._repeat)
```

Comment:
Per-offline discussion with Haichen, there is no need to divide by three because each result is the mean of three runs.